sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -368,6 +368,24 @@ object functions {
    */
   def collect_set(columnName: String): Column = collect_set(Column(columnName))
 
+  /**
```

Comment:
Returns a count-min sketch of a column with the given esp,
      confidence and seed. The result is an array of bytes, which can be deserialized to a
      `CountMinSketch` before usage. Count-min sketch is a probabilistic data structure used for
      cardinality estimation using sub-linear space.

Is the comment above more aligned?

https://github.com/apache/spark/blob/00e7c08606d0b6de22604d2a7350ea0711355300/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/CountMinSketchAgg.scala#L45-L48

Reply:
Thanks, done.